docs/getting-started/projectsetup/java_api_quickstart.md

```diff
@@ -336,10 +333,7 @@ can run the application from the JAR file without additionally specifying the ma
 Write your application!
 
 If you are writing a streaming application and you are looking for inspiration what to write,
-take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/tutorials/datastream_api.html#writing-a-flink-program).
-
-If you are writing a batch processing application and you are looking for inspiration what to write,
-take a look at the [Batch Application Examples]({{ site.baseurl }}/dev/batch/examples.html).
+take a look at the [Stream Processing Application Tutorial]({{ site.baseurl }}/getting-started/tutorials/datastream_api.html#writing-a-flink-program).
```

Comment:
We will remove the Tutorials as part of this FLIP. So in the future these links should probably only point examples and code walk throughs. What do you think?